catalyst/dl/core/runner.py

```diff
@@ -166,11 +169,17 @@ def _run_epoch(self, loaders):
                 self._run_loader(loader)
             self._run_event("loader_end")
 
+    def _run_prestage(self, stage: str):
```

Comment:
looks like a dirty hack :)

Reply:
I created this method to memorize something in runner before stage begins but after initialization is done. It's obviously a little bit hacky, but I think it's still useful, i.e. the hack is not dirty=)